_scripts/gh-pages.sh

```diff
@@ -12,13 +12,8 @@ echo "Building docs."
 cd packages/fxa-email-service
 cargo doc --no-deps
 
-# fxa-payments-server relies on fxa-content-server .scss styles, which in turn
-# rely on some modules in package.json
-cd ../../packages/fxa-content-server
-npm ci
-
 cd ../../packages/fxa-payments-server
-npm ci
+yarn workspaces focus fxa-payments-server
```

Comment:
Hmm, does this break anything in the storybook build? IIRC, part of the `npm ci` over in content-server for this particular thing is the post-install stuff over there that payments-server needs for its own build

Will likely replace this with PR #5357 - but I'm doing this same `npm ci` there too

Reply:
"postinstall" will still run on content-server since payments-server has it as a dependency